libs/db-browser/src/main/java/com/oceanbase/tools/dbbrowser/editor/mysql/MySQLColumnEditor.java

```diff
@@ -130,6 +130,9 @@ public void appendModifier(DBTableColumn column, SqlBuilder sqlBuilder) {
                 if (isDefaultValueBuiltInFunction(column)) {
                     // 默认值是 内置函数，则不需要用引号转义，直接拼接即可
                     sqlBuilder.append(defaultValue);
+                } else if (defaultValue.equals("(empty_string)")) {
```

Comment:
what dose `(empty_string)` mean? Is it equals to `defaultValue.isEmpty()`?
 and please add a ut test for it.